pkg/controller/common/annotation/controller_version.go

```diff
@@ -15,7 +15,11 @@ import (
 )
 
 // ControllerVersionAnnotation is the annotation name that indicates the last controller version to update a resource
-const ControllerVersionAnnotation = "common.k8s.elastic.co/controller-version"
+const (
+	ControllerVersionAnnotation       = "common.k8s.elastic.co/controller-version"
+	MinCompatibleControllerVersion    = "0.10.0-SNAPSHOT"
```

Comment:
I created https://github.com/elastic/cloud-on-k8s/issues/1833 so we fix the version number as a follow-up PR.

Reply:
Thanks.